src/pages/Users/helper.tsx

```diff
@@ -0,0 +1,28 @@
+import { BaseData } from '@/components/UniversalTable/types'
+
+export type UserData = {
+	id: string | number;
+	[key: string]: any; // інші поля можуть бути будь-якими
+  };
+
+export const mapDataUserPage = (usersDataResponse: UserData[]) => {
+	const tableData = {
+		header: 'Users',
+		data: {
+			headers: Object.keys(usersDataResponse[0]).map(key => {
+				const title = key.split('')
+				title[0] = title[0].toUpperCase()
+				const header = {
+					title: title.join(''),
+					property: key,
+				}
+				return header
+			}),
+			data: usersDataResponse.map(user => {
```

Comment:
too many `data`...I won't use this function. if I'd see it somewhere used I'd remove it completely and write a new one instead. 

so just get rid of this unnecessary nesting tons of objects inside each other